packages/core-contracts/test/proxy/ForwardingProxy.test.js

```diff
@@ -13,7 +13,8 @@ describe('ForwardingProxy', () => {
       Implementation = await factory.deploy();
 
       factory = await ethers.getContractFactory('ForwardingProxyMock');
-      Proxy = await factory.deploy(Implementation.address);
+      Proxy = await factory.deploy();
+      await Proxy.initialize(Implementation.address);
```

Comment:
I'd recommend to remove constructors in a different PR.

If you still want to do this, consider that this returns a tx that needs to be awaited.
```const tx = await Proxy.initialize(...)
await tx.wait()```